.gitignore

```diff
@@ -51,3 +51,14 @@ a.out.dSYM/
 
 conv_3_1.o
 include/verilogAST.hpp
+tests/unit/bit_replacement.json
```

Comment:
are these gold files? might be easier to put them into `tests/gold/` and then ignore `tests/gold/*`

Reply:
Added task in CoreIR cleanup to do the build/ gold/ flow for tests.